### ISSUE TITLE
passifox is always requesting passwords

### ISSUE THREAD

**Issue by moozer** [pfn/passifox#520]
I updated to passifox version 1.2.0, and there is a new behavior.

Whenever I go to a new page, passifox want to check if there are passwords available - this is quite annoying, since my database i usually locked, and the keepass window  pops in front.

Can I disable that passifox talks to the password database before I explicitely asks it too?

**Comment by pfn** [comment]
The behavior has never changed, no.

**Comment by moozer** [comment]
So passifox, by design, queries the database for every page that gets loaded?

**Comment by pfn** [comment]
For every page that has a password field

**Comment by pfn** [comment]
That's not entirely correct, it is a password storage backend for Firefox's
password manager, it requests everytime Firefox asks, which is usually only
when there is a password field

**Comment by moozer** [comment]
So it might be related to the debian shift from iceweasel to firefox...
I'll check firefox's settings.

**Comment by cloutierjo** [comment]
Did you find a solution for this? I'm facing this problem, but i'm new to passiFox so I cant compare to "before"

**Comment by moozer** [comment]
no, stille having the issue. I suspect my installation of Debian + firefox to be the culprit, it is not only firefox behaving oddly....

**Comment by agsteele** [comment]
This issue has also appeared in #555 and a solution identified.  A pr has been proposed in #564
